react/week1/todos/src/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+// import App from './App';
+// import * as serviceWorker from './serviceWorker';
+
+
+const root = document.getElementById('root')
+
+ function TodoItem(props) {
+   return <li>{props.description}, {props.deadline}</li>
+ }
+
+class Todo extends React.Component {
```

Comment:
You could use a better naming here such as TodoList. It's always nice to look at the component and to know immediately what it is doing.